--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -13,6 +13,8 @@
     <ExternalConsole>false</ExternalConsole>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <Nullable>disable</Nullable>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
+    <NoWarn>$(NoWarn);NU1901</NoWarn>
     <Product>OpenRA</Product>
     <Copyright>Copyright (c) The OpenRA Developers and Contributors</Copyright>
   </PropertyGroup>
@@ -32,11 +34,11 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <Optimize>false</Optimize>
     <!-- Enable only for Debug builds to improve compile-time performance for Release builds -->
-    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <EnforceCodeStyleInBuild>false</EnforceCodeStyleInBuild>
     <!-- Enabling GenerateDocumentationFile is required for IDE0005 (Remove unnecessary import)
       rule to run in command line builds. https://github.com/dotnet/roslyn/issues/41640
       Enable only for Debug builds to improve compile-time performance for Release builds -->
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
   </PropertyGroup>

   <ItemGroup>
