--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -13,6 +13,8 @@
     <ExternalConsole>false</ExternalConsole>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <Nullable>disable</Nullable>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
+    <NoWarn>$(NoWarn);NU1901</NoWarn>
     <Product>OpenRA</Product>
     <Copyright>Copyright (c) The OpenRA Developers and Contributors</Copyright>
   </PropertyGroup>
